.jscsrc

```diff
@@ -3,6 +3,7 @@
   "requireCamelCaseOrUpperCaseIdentifiers": "ignoreProperties",
   "validateIndentation": "\t",
   "disallowMultipleSpaces": true,
+  "disallowQuotedKeysInObjects": "allButReserved",
```

Comment:
This is what I was looking for before - allows certain properties to be quoted such as 'default'.